src/gtnh/gui.py

```diff
@@ -17,6 +18,9 @@
 from gtnh.pack_downloader import download_mod, ensure_cache_dir
 from gtnh.utils import get_latest_release, get_token, load_gtnh_manifest, sort_and_write_modpack
 
+LOGGER = logging.getLogger("gui")
```

Comment:
nit: all caps should be constants, I'd prefer `log =` and use `log.info`

Reply:
Should be fixed